src/components/organisms/AppRouter/AppRouter.jsx

```diff
@@ -57,7 +57,7 @@ const AppRouter = () => {
   return (
     <Router basename="/">
       <Switch>
-        <Route path="/SparkleVerse" component={SparkleSpaceMarketingPage} />
+        <Route path="/SparkleVerse" component={() => <Redirect to="https://sparklever.se/" />} />
```

Comment:
could we put the URL in a settings.ts? That way it can be referenced throughout the app